### PR TITLE
Gac 005 aiqn training

### PR DESCRIPTION
**Pull request by gwbcho**
Cleanup.

### CODE REVIEW

File: policies/policy.py [hunk 1]
```diff
@@ -1,38 +1,56 @@
...
 import numpy as np
 import tensorflow as tf
 
 # internal dependencies
-import policies.policy_helpers.helpers as helpers
-import policies.policy_helpers.helper_classes as helper_classes
 import utils.utils as utils
 
+# import local dependencies
+import policies.gac.gac_networks as gac_networks
```
Comment (axlbrown) [inline policies/policy.py]: policies.gac.gac_networks ---> policies.gac.networks

**Review by axlbrown — CHANGES_REQUESTED** [review]

File: policies/policy.py (hunk 1 above)
Comment (gwbcho) [reply policies/policy.py]: Didn't notice that you'd changed the file name.

**Comment by axlbrown** [comment]
Yeah. I changed it this evening.

**Review by axlbrown — APPROVED** [review]
lgtm

**Pull request merged**